src/components/Navbar/index.tsx

```diff
@@ -5,32 +5,56 @@ import {
   useAuthStateContext
 } from "../../context/auth";
 import { Select } from "antd";
+import { useQuery } from "@apollo/react-hooks";
+import {
+  GET_PROJECTS,
+  ProjectsQuery,
+  Project
+} from "graphql/queries/get-projects";
 
 const { Option, OptGroup } = Select;
 
+const renderProjectOption = (isFavorite: boolean = false) => ({
+  identifier,
+  displayName
+}: Project) => (
+  // two Options cannot have the same value attribute or it breaks ability to scroll with keyboard.
+  // therefore "-favorite" is appended to the end of a favorite's value prop
+  <Option
+    key={identifier}
+    value={isFavorite ? `${identifier}-favorite` : identifier}
```

Comment:
You could run into the same problem on the off-chance someone names a project xxx-favorite. You should have an escape sequence or prevent this scenario in the first place (below)